### PR TITLE
DevTools > Search

### PR DESCRIPTION
**Pull request by sofiayem**
Documented:
- [Missing] Search tab
- M98 > [Search tab shortcuts](https://developer.chrome.com/blog/new-in-devtools-98/#toggle-search-result)
- Search bars built into several tools

### CODE REVIEW

File: site/en/docs/devtools/search/index.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
+---
+layout: "layouts/doc-post.njk"
+title: "Search: Find text across all loaded resources"
+authors:
+  - sofiayem
+date: 2022-05-27
```
Comment (malchata) [inline site/en/docs/devtools/search/index.md:6]:
```suggestion
date: 2022-06-01
```

File: site/en/docs/devtools/search/index.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+{% endAside %}
+
+Alternatively, use [built-in search bars](#built-in-search-bars) to find text in a resource opened in a specific tool.
+
+## Open the Search tab {: #open-search }
+
+You can open the **Search** tab in several ways. First, [open DevTools](/docs/devtools/open/), then do one of the following:
+
+- In the top-right corner of DevTools, select {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/N5Lkpdwpaz4YqRGFr2Ks.svg", alt="Customize and control DevTools", width="20", height="20" %} **Customize and control DevTools** > **More tools** > **Search**.
+- Press `Esc` to open [**Drawer**](/docs/devtools/customize/#drawer), and in the top-left corner, select {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/N5Lkpdwpaz4YqRGFr2Ks.svg", alt="Customize and control DevTools", width="20", height="20" %} **More Tools** > **Search**.
```
Comment (malchata) [inline site/en/docs/devtools/search/index.md:24]: I wonder if we can use the `<kbd>` element here for better semantics? Or maybe we don't style that appropriately on d.c.c. Just a thought.

Comment (malchata) [inline site/en/docs/devtools/search/index.md:24]:
```suggestion
- Press <kbd>Esc</kbd> to open [**Drawer**](/docs/devtools/customize/#drawer), and in the top-left corner, select {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/N5Lkpdwpaz4YqRGFr2Ks.svg", alt="Customize and control DevTools", width="20", height="20" %} **More Tools** > **Search**.
```

File: site/en/docs/devtools/search/index.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+
+{% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/D3YatkHJm3NkOEPMcxzg.png", alt="Search results.", width="800", height="626" %}
+
+To make your query case-sensitive, toggle the {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/egjnpBbgTvj6FDiIbfoc.png", alt="Match case.", width="25", height="20" %} **Match Case** button.
+
+{% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/u2YlZUWEgtSIr7Ma7iol.png", alt="Match Case button.", width="800", height="444" %}
+
+To search for text that matches a RegEx pattern, toggle the {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/97kuRQETaw1jnAfMHrbQ.png", alt="RegEx button.", width="17", height="18" %} **Use Regular Expression** button.
+
+{% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/OMfgsPc0jnodtsmaL7A1.png", alt="RegEx button.", width="800", height="558" %}
```
Comment (malchata) [inline site/en/docs/devtools/search/index.md:39]:
```suggestion
<figure>
  {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/OMfgsPc0jnodtsmaL7A1.png", alt="RegEx button.", width="800", height="558" %}
</figure>
```

**Comment by malchata** [comment]
@sofiayem, feel free to merge whenever you're ready. I had some small nits, nothing much.

**Review by jecfish — APPROVED** [review]
LGTM % comments

**Pull request merged**